library/qs_source/core/core.qs

```diff
@@ -56,4 +56,7 @@ namespace Microsoft.Quantum.Core {
 
         output
     }
+
+    export Length,
+    Repeated;
```

Comment:
I think once your formatting change arrives this might need to be indented. Also, is it worth considering moving these to the top of the namespaces so when the file is opened fresh someone can see all the exports? That might get distracting for the longer lists, but would still be informative. Your call.

Reply:
Actually, never mind on moving these to the top. I realize now the exports are effectively for the purpose of making the callables "public" and later we'll have a std somewhere that re-exports things.